src/row.rs

```diff
@@ -60,7 +60,7 @@ impl Row {
         let (mut cx, mut rx) = (0, 0);
         for c in String::from_utf8_lossy(&self.chars).chars() {
             // The number of bytes used to store the character
-            let n_bytes = c.len_utf8();
+            let n_bytes = if c == '\t' { tab - (rx % tab) } else { c.len_utf8() };
```

Comment:
I don't think this is right. If you type `<tab>` `a`, the cursor will misplaced (it cannot be moved to the `a` or after, even after pressing the `End` key).

Reply:
Indeed.  Need to rework this fix.